assignment2/javascript/gridGame.js

```diff
@@ -0,0 +1,18 @@
+function changeColor(element) {
+  if(document.getElementById("red").checked) {
+    element.style.background = "red";
+  }
+  else if(document.getElementById("blue").checked) {
+    element.style.background = "blue";
+  }
+  if(document.getElementById("green").checked) {
+    element.style.background = "green";
+  }
+}
```

Comment:
What if there were 10 colors? 10 if else? instead of writing if-else, simply get the element which is selected and get it's value.